hack/install/install_criu.sh

```diff
@@ -48,6 +48,7 @@ main() {
   os_dist="$(detect_os)"
   if [[ "${os_dist}" = "Ubuntu" ]]; then
     criu::ubuntu::install_dependencies > /dev/null
+    echo ">>>> start to download criu from github repository <<<<"
```

Comment:
If the downloading task fails, would this script show the detailed reason why it fails? @Ace-Tang

Reply:
I think yes.

Reply:
only know we stack at downing from github.

Reply:
ok, then it meets my demand.